### PR TITLE
(GH-932) Add instructions how to share code

### LINKED ISSUE #932: Add documentation for sharing of code
We should have documentation which shows the different possibilities to share code across multiple repos (Recipe scripts, addins, modules, ..)

### PR DESCRIPTION
**Pull request by pascalberger**
Add instructions how to reuse code across projects and repositories

Fixes #932

### CODE REVIEW

File: input/docs/writing-builds/reusing-builds.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Description: How to reuse build code in different projects and repositories
+---
+
+Cake supports different ways to share build code across multiple projects and repositories.
+
+| Method                         | Sharing of classes                              | Sharing of aliases                              | Sharing of tasks                                |
+|--------------------------------|-------------------------------------------------|-------------------------------------------------|-------------------------------------------------|
+| [Recipe NuGet packages]        | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-times" style="color:red"></i>   | <i class="fa fa-check" style="color:green"></i> |
+| [Addins]                       | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-times" style="color:red"></i>   |
+| [Assemblies in NuGet packages] | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-times" style="color:red"></i>   | <i class="fa fa-times" style="color:red"></i>   |
```
Comment (gep13) [inline input/docs/writing-builds/reusing-builds.md:11]: Are we missing a link for this one?

**Review by gep13 — CHANGES_REQUESTED** [review]

**Review by gep13 — APPROVED** [review]
LGTM!

File: input/docs/writing-builds/reusing-builds.md (hunk 1 above)
Comment (pascalberger) [reply input/docs/writing-builds/reusing-builds.md:11]: This actually works due to some magic in Wyam. But I added link explicitely.

Comment (gep13) [reply input/docs/writing-builds/reusing-builds.md:11]: Gotcha.  I think adding it explicitly makes a lot of sense.

**Comment by gep13** [comment]
@pascalberger your changes have been merged, thanks for your contribution 👍

**Pull request merged**